portfolio/src/main/webapp/index.html

```diff
@@ -15,7 +16,7 @@
         <li><a href="#contact">Contact</a></li>
         <li><a href="#comments">Comments</a></li>
     </ul>
-    <div id="content">
+    <div class="non-map" id="content">
```

Comment:
If this `non-map` class exists just for easier selection in CSS, you could consider using the `:not` pseudo-selector, instead: https://developer.mozilla.org/en-US/docs/Web/CSS/:not

Reply:
For this feature, I think I will discuss it with you in the afternoon 1:1 meeting. I follow the instructions but the css file still mess my map up and I could not figure out the reason.